include/motor-control/core/motor_interrupt_handler.hpp

```diff
@@ -163,17 +193,7 @@ class MotorInterruptHandler {
     void finish_current_move() {
         has_active_move = false;
         tick_count = 0x0;
-        if (buffered_move.group_id != NO_GROUP) {
-            auto ack = Ack{.group_id = buffered_move.group_id,
-                           .seq_id = buffered_move.seq_id,
-                           .current_position = static_cast<uint32_t>(
-                               position_tracker >>
-                               31),  // TODO (AA 2021-11-10): convert
-                                     // this value to mm instead of steps
-                           .ack_id = AckMessageId::complete};
-            static_cast<void>(
-                status_queue_client.send_move_status_reporter_queue(ack));
-        }
+        send_ack(false, AckMessageId::complete);
```

Comment:
Want to use the value passed into `finish_current_move` rather than setting it to false always.